pkg/util/util.go

```diff
@@ -104,10 +106,17 @@ func Forever(f func(), period time.Duration) {
 }
 
 // Until loops until stop channel is closed, running f every period.
+// Until is syntactic sugar on top of JitterUntil with zero jitter factor
+func Until(f func(), period time.Duration, stopCh <-chan struct{}) {
+	JitterUntil(f, period, 0.0, stopCh)
+}
+
+// JitterUntil loops until stop channel is closed, running f every period.
+// If jitterFactor is positive, the period is jittered before every run of f.
```

Comment:
JitterUntil could check if the period is negative and return immediately if so. So the same test as for closed channel would apply. Imho, both ways are legit. On the other hand if you specify negative jitter by accident, you don't blow up entire sync loop if it is ignored. Instead, the JitterUntil could log error/warning the factor is negative.